datafusion/spark/src/function/bitmap/bitmap_count.rs

```diff
@@ -171,4 +192,31 @@ mod tests {
         );
         Ok(())
     }
+
+    #[test]
+    fn test_dictionary_encoded_bitmap_count_invoke() -> Result<()> {
```

Comment:
Could we add this as an SLT instead? Can make use of `arrow_cast(value, 'Dictionary(...)')` to ensure the values are dictionaries